Duality/Resources/Font.cs

```diff
@@ -876,14 +890,15 @@ public int PickTextGlyph(string text, float x, float y)
 			return -1;
 		}
 
-		private void ProcessTextAdv(string text, int index, out GlyphData glyphData, out Rect uvRect, out float glyphXAdv, out float glyphXOff)
+		private void ProcessTextAdv(string text, int index, out GlyphData glyphData, out Rect uvRect, out float glyphXAdv, out float glyphXOff, out float glyphYOff)
 		{
 			char glyph = text[index];
 			int charIndex = (int)glyph > this.charLookup.Length ? 0 : this.charLookup[(int)glyph];
 			this.texture.LookupAtlas(charIndex, out uvRect);
 
 			this.GetGlyphData(glyph, out glyphData);
 			glyphXOff = -glyphData.OffsetX;
+			glyphYOff = glyphData.OffsetY;

```

Comment:
Can you explain here a bit more. As far as I can see offset x and offset y are purely an  offset within the cell of the character block (ie the cell might be padded or the cells might be a constant width and height) so in all the cases I can't see how they would ever be negative - both x and y. Kernings are actually used for manipulating the x-axis to bring then closer but otherwise I can only see offset x and y being positive.